test_sell_today.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from data_analysis.profit_analysis import sell_today
+
+
+@pytest.mark.parametrize("a,b,expect", [(0.1, 100, 0), (100, 0.1, 1)])
+def tests_sell_today(a, b, expect):
+    assert sell_today(a, b, 1, 1) == expect
+
+
+def main():
```

Comment:
You don't have to do this, pytest will run the tests if you follow the naming convention as you have above

Reply:
so i dont need either the assert line or to have a main function ?